packages/account/src/Components/additional-kyc-info-modal/additional-kyc-info-form.tsx

```diff
@@ -37,18 +37,19 @@ type TAdditionalKycInfoFormProps = {
 
 export const AdditionalKycInfoForm = observer(({ setError }: TAdditionalKycInfoFormProps) => {
     const { client, ui, notifications } = useStore();
-    const { account_settings, residence_list } = client;
-    const { fields, initialValues, validate } = getFormConfig({
-        account_settings,
-        residence_list,
-        required_fields: ['place_of_birth', 'tax_residence', 'tax_identification_number', 'account_opening_reason'],
-    });
-
+    const { residence_list, updateAccountStatus } = client;
     const {
         update,
         mutation: { isLoading, isSuccess, error, isError },
+        data,
```

Comment:
`data: account_settings_data`

Better readability if we are passing it around, so that the next person working on this knows it without going backwards to find the source of truth

Reply:
Fair point @thisyahlen-deriv 
Updated 👍